Formula/f/fernflower.rb

```diff
@@ -0,0 +1,23 @@
+class Fernflower < Formula
```

Comment:
Why is it called fernflower?

Reply:
sorry, I missed the github the url, I thought it just has the `java-decompiler` and sonehow needs to be inferred from the build.